platform-friends-hybrid/styles/main.css

```diff
@@ -32,585 +46,610 @@
+  border: 0 none;
+  background-color: transparent;
 }
 
 /* No API Key screen */
 
+#friendsApp .noapikey-scrn h3 {
+  line-height: 1.25;
+}
+
 #friendsApp .noapikey-scrn p {
-    line-height: 1.5;
-    color: #e4e4e4;
+  line-height: 1.5;
 }
 
 #friendsApp .noapikey-scrn p strong,
 #friendsApp .noapikey-scrn p span {
-    color: #fff;
+  color: #000;
 }
 
 #friendsApp .noapikey-scrn p span {
-    font-weight: 600;
+  font-weight: 600;
 }
 
 /* Signup */
 
-#friendsApp .signup-view .select-wrp {
-    -webkit-border-radius: 0 0 2px 2px;
-    -moz-border-radius: 0 0 2px 2px;
-    border-radius: 0 0 2px 2px;
-}
-
-#friendsApp.km-flat .signup-view .km-listinset > li:last-child {
-    margin: .5em 0 0;
-    -webkit-border-radius: 2px;
-    -moz-border-radius: 2px;
-    border-radius: 2px;
-}
-
-#friendsApp .txtarea-wrp textarea {
-    float: none;
-    height: 5.5em;
-    resize: none;
+#friendsApp.km-flat textarea {
+  float: none;
+  height: auto;
+  margin: 0;
+  resize: none;
 }
 
 #friendsApp .signup-view .signup-btn-wrp {
-    margin: 0 1em;
+  margin: 0 2em;
 }
 
 #friendsApp .signup-view .select-wrp select {
-    color: #b6c5c6;
+  color: #4d4d4d;
 }
 
 #friendsApp .signup-view .select-wrp option:first-child {
-    color: #b6c5c6;
+  color: #b6c5c6;
 }
 
 /* Activities */
 
-#friendsApp .activities-view .km-content,
-#friendsApp .activity-view .km-content,
-#friendsApp .add-activity-view .km-content,
-#friendsApp .add-comment-view .km-content,
-#friendsApp .activities-view .km-listview {
-    background-color: #fff;
+#friendsApp .activity-view .user-status-wrp .km-navbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  padding-top: .5em;
+  background-color: transparent;
+}
+
+#friendsApp .activity-view .user-status-wrp .content {
+  padding: 1em;
+}
+
+#friendsApp .activity-view .activity-content {
+  padding: 0;
+}
+
+#friendsApp .activity-view .activity-content > span {
+  padding: 1em 0;
+}
+
+#friendsApp .activity-view .user-info {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 2;
+  width: 100%;
+  padding: 0 1em .5em;
+}
+
+#friendsApp .activity-view .user-info .person-name,
+#friendsApp .activity-view .user-info .time-span {
+  color: #fff;
+}
+
+#friendsApp .activity-view .user-info .person-name {
+  display: inline-block;
+  float: left;
+  margin: 1em;
+}
+
+#friendsApp .activity-view .user-info .time-span {
+  display: inline-block;
+  float: right;
+  min-width: 80px;
+  margin: 1.65em 0 0 0;
+  font-size: .85em;
+  text-align: right;
+  text-transform: uppercase;
 }
 
 #friendsApp .activities-view .km-list > li,
 #friendsApp .activity-view .km-list > li {
-    overflow: visible;
-    padding: .75em 0;
-    border-width: 0 0 1px;
-    border-color: #eaeced;
-    background-color: #fff;
-    -webkit-box-shadow: none;
-    box-shadow: none;
-    color: #34495e;
+  padding: .75em 0 1em;
+  border-width: 0;
+  color: #4d4d4d;
+  background-color: #fff;
+  -webkit-box-shadow: none;
```

Comment:
We can probably get away without prefix here too, unless we still support Android 2.3:
http://caniuse.com/#search=box-shadow
https://developer.android.com/about/dashboards/index.html - 6% only lately.